Document/0x05b-Basic-Security_Testing.md

```diff
@@ -714,11 +714,43 @@ Other resources where you might find useful information are:
 
 ##### Xposed
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1234>
-
 [Xposed](http://repo.xposed.info/module/de.robv.android.xposed.installer) is a "framework for modules that can change the behavior of the system and apps without touching any APKs." Technically, it is an extended version of Zygote that exports APIs for running Java code when a new process is started. Running Java code in the context of the newly instantiated app makes it possible to resolve, hook, and override Java methods belonging to the app. Xposed uses [reflection](https://docs.oracle.com/javase/tutorial/reflect/ "Reflection Tutorial") to examine and modify the running app. Changes are applied in memory and persist only during the process' run times—no patches to the application files are made.
 
-To use Xposed, you need to first install the Xposed framework on a rooted device. Deploy modifications deployed in the form of separate apps ("modules"), which can be toggled on and off in the Xposed GUI.
+To use Xposed, you need to first install the Xposed framework on a rooted device as explained on [XDA-Developers XPosed framework hub](https://www.xda-developers.com/xposed-framework-hub/). Deploy modifications deployed in the form of separate apps ("modules"), which can be toggled on and off in the Xposed GUI.
```

Comment:
Add alt text to URL

> Deploy modifications deployed in the form of separate apps ("modules"), which can be toggled on and off in the Xposed GUI.

Probably to the following (there is also overlap with previous paragraph):

> Modules can be installed through the Xposed Installer app, and they can be toggled on and off through GUI.